README.md

```diff
@@ -34,21 +34,30 @@ workflow/
 ```
 
 
-## Running Example Analyis
+## Running An Analyis
 
 Brieflow is set up as a Snakemake workflow with user configuration between steps where necessary. 
 Thus, a user must configure parameters between workflow steps with configuration notebooks.
-This example analysis details the steps necessary for configuring parameters and running step workflows.
 While each step's workflow has its own Conda environment (compiled by Snakemake at runtime), the notebooks all share a configuration environment.
 
-### Step 1: Set up workflow/configuration Conda environments
+We currently recommend creating a cloned version of Brieflow for each screen analysis with:
+```sh
+# change directory below to reflect location of a screen analysis project
+cd screen_analysis_dir/
+git clone https://github.com/cheeseman-lab/brieflow.git
+```
+
+See the steps below to set up the workflow/configuration environments and run your own analysis with Brieflow.
```

Comment:
I would add that we will soon release documentation on how to set up an analysis repo for working with brieflow!

Reply:
Done in https://github.com/cheeseman-lab/brieflow/pull/28/commits/d2abbe5c2ac1a1c9fb5cadb8fefb50b59533d162